### PR TITLE
COP flow fullname change

### PR DESCRIPTION
**Pull request by Karthik-TF**

### CODE REVIEW

File: pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/service/NotificationService.java [hunk 1]
```diff
@@ -125,6 +125,9 @@ public class NotificationService {
 	@Value("${preregistration.identity.name}")
 	private String fullName;
 
+	@Value("${preregistration.identity.name}")
+	private String defaultFullName;
```
Comment (SowmyaGoudar) [inline pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/service/NotificationService.java:126]: Please remove this and update the above existing config name to defaultFullName

File: pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/service/NotificationService.java [hunk 2]
```diff
@@ -385,6 +388,15 @@ private List<KeyValuePairDto<String, String>> getLangaueNamePairs(JsonNode respo
 		List<KeyValuePairDto<String, String>> langaueNamePairs = new ArrayList<KeyValuePairDto<String, String>>();
 		List<KeyValuePairDto<String, String>> langaueNamePairsfullName = new ArrayList<KeyValuePairDto<String, String>>();
 		KeyValuePairDto<String, String> langaueNamePair = null;
+		String update = "UPDATE";
+		String userService = responseNode.get("userService").toString();
+		userService = userService.substring(1, userService.length() - 1);
+		if(userService.equalsIgnoreCase(update)){
+			fullName="surnameCop";
```
Comment (SowmyaGoudar) [inline pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/service/NotificationService.java]: Please add new configuration for this

Comment (SowmyaGoudar) [reply pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/service/NotificationService.java]: if(userService.equalsIgnoreCase(update)){
			fullName="surnameCop";
		}
		else {
			fullName=defaultFullName;
		}
		for (String name : fullName.split(",")) {

**Review by SowmyaGoudar — COMMENTED** [review]
Please fix the review comments

File: pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/service/NotificationService.java (hunk 1 above)
Comment (Karthik-TF) [reply pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/service/NotificationService.java:126]: Removed mam and changed the existing one to defaultFullName

File: pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/service/NotificationService.java (hunk 2 above)
Comment (Karthik-TF) [reply pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/service/NotificationService.java]: added if(userService.equalsIgnoreCase(update)){
			fullName=defaultFullNameCop;
		}
		else {
			fullName=defaultFullName;
		}

**Pull request merged**